### PR TITLE
Tamil Config and Translation files updated

### PR DESCRIPTION
**Pull request by TamilNeram**
Added only Tamil config file and the translation file.

Fixes #

## Proposed changes

- 
- 
- 

## Test methodology 

- 
- 
- 

## Test environment(s) 

- GIT 
- Windows 

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by gerhardol** [comment]
Flag still needed

**Comment by TamilNeram** [comment]
Tamil
Tamil universal flag

**Comment by RussKie** [comment]
The setup configurations must be updated for the build to pass:
```
  [ERROR] Items not declared in Product.wxs:
  	C:\projects\gitextensions\artifacts\Release\publish\GitExtensions\Translation\Tamil.gif
  	C:\projects\gitextensions\artifacts\Release\publish\GitExtensions\Translation\Tamil.Plugins.xlf
  	C:\projects\gitextensions\artifacts\Release\publish\GitExtensions\Translation\Tamil.xlf
C:\projects\gitextensions\GitExtensions\Project.Publish.targets(218,5): error MSB3073: The command "..." exited with code 1. [C:\projects\gitextensions\GitExtensions\GitExtensions.csproj]
```

```diff
diff --git a/Setup/Product.wxs b/Setup/Product.wxs
index 91aaa62c0..122ba3fd6 100644
--- a/Setup/Product.wxs
+++ b/Setup/Product.wxs
@@ -526,6 +526,15 @@
       <Component Id="Spanish.Plugins.xlf" Guid="*">
         <File Source="$(var.ArtifactsPublishPath)\Translation\Spanish.Plugins.xlf" />
       </Component>
+      <Component Id="Tamil.gif" Guid="*">
+        <File Source="$(var.ArtifactsPublishPath)\Translation\Tamil.gif" />
+      </Component>
+      <Component Id="Tamil.xlf" Guid="*">
+        <File Source="$(var.ArtifactsPublishPath)\Translation\Tamil.xlf" />
+      </Component>
+      <Component Id="Tamil.Plugins.xlf" Guid="*">
+        <File Source="$(var.ArtifactsPublishPath)\Translation\Tamil.Plugins.xlf" />
+      </Component>
       <Component Id="TraditionalChinese.gif" Guid="*">
         <File Source="$(var.ArtifactsPublishPath)\Translation\Traditional Chinese.gif" />
       </Component>
@@ -844,6 +853,11 @@
           <ComponentRef Id="Spanish.Plugins.xlf" />
           <ComponentRef Id="Spanish.gif" />
         </Feature>
+        <Feature Id="Tamil" Title="Tamil" Level="1">
+          <ComponentRef Id="Tamil.xlf" />
+          <ComponentRef Id="Tamil.Plugins.xlf" />
+          <ComponentRef Id="Tamil.gif" />
+        </Feature>
         <Feature Id="TraditionalChinese" Title="Traditional Chinese" Level="1">
           <ComponentRef Id="TraditionalChinese.xlf" />
           <ComponentRef Id="TraditionalChinese.Plugins.xlf" />
```

Run:
```powershell
pushd .\.tools\transifex\
.\DownloadTransifexTranslations.ps1
popd
git stage Setup/Product.wxs
```

**Comment by TamilNeram** [comment]
updated the product.wxs file

File: Setup/Product.wxs [hunk 1]
```diff
@@ -445,23 +445,32 @@
       </Component>
     </DirectoryRef>
     <DirectoryRef Id="TranslationsDir">
-      <Component Id="English.gif" Guid="*">
```
Comment (gerhardol) [inline Setup/Product.wxs:448]: Please keep the entries in alphabetic order, hard to see differences now

**Review by gerhardol — COMMENTED** [review]
Flag should be fine. Meaning: https://medium.com/@prasanna_says/2000-years-ago-a-tamil-poet-wrote-yaadhum-oore-yaavarum-kelir-to-us-all-towns-are-one-all-dc87f56b8cfc

**Comment by RussKie** [comment]
> updated the product.wxs file

That's not the result of the script execution

**Comment by TamilNeram** [comment]
Sorted the file content with two more commits. You can find the diff is clean with only addition of Tamil file entries.

**Review by RussKie — APPROVED** [review]

**Review by mstv — APPROVED** [review]

**Review by gerhardol — APPROVED** [review]

**Pull request merged**